### DISCUSSION TITLE
Alternative raspberry

### DISCUSSION THREAD

**Discussion by FrancescoDalan** [indomus/forum#2034] [HUB: Home Assistant]
Ciao a tutti,
Con i costi di Raspberry pi4 alle stelle, vorrei cercare delle alternative più economiche per installare Homeassistant. 

Ho provato diversi TVbox ma, con Android 10, non sono riuscito a sbloccare il boot loader per bootare da SD (non sono nemmeno riuscito a downgradare a Android 9). Qualche consiglio?

grazie mille a tutti

**Comment by KC-inDomus** [comment]
> Discussion options

Leggi qui: https://indomus.it/componenti/mini-pc-intel-nuc-e-similari/
Più economiche è difficile (se parli del prezzo di due anni fa), ma molto vicine o allineate ce ne sono svariate.

**Reply by FrancescoDalan** [reply]
Grazie mille,
Buona giornata
